packages/react/src/primitives/Collection/__tests__/Collection.test.tsx

```diff
@@ -6,9 +6,8 @@ import kebabCase from 'lodash/kebabCase';
 import { Collection } from '../Collection';
 import { Flex } from '../../Flex';
 import { Text } from '../../Text';
+import { ComponentClassNames, ComponentText } from '../../shared/constants';
 import { ComponentPropsToStylePropsMap } from '../../types';
-import { ComponentText } from '../../shared/constants';
-import { ComponentClassName } from '../../shared/types';
```

Comment:
This was the only test that was using `ComponentClassName` instead of `ComponentClassNames`. Updated to use the latter.

Reply:
Might be out of scope for this PR, but pretty we can just delete `ComponentClassNames` and replace usage directly with `ComponentClassName`

Reply:
+1, we have some non-official exports from `@aws-amplify/ui-react` that should be cleaned up. Out of scope for this PR however.